src/main.rs

```diff
@@ -0,0 +1,326 @@
+use std::{
+    env,
+    error::Error,
+    ffi::OsString,
+    fs::File,
+    io::{self, Write},
+    process::ExitCode,
+};
+
+use pgxn_meta::Validator;
+use serde_json::Value;
+
+// Minimal main function; logical is all in run.
+fn main() -> Result<ExitCode, Box<dyn Error>> {
+    run(io::stdout(), env::args_os().collect::<Vec<_>>())
```

Comment:
```suggestion
    run(io::stdout(), env::args_os())
```

Can likely get away from `collect`ing here since `args_os` already returns an iterator

Reply:
Oh, thanks! Done in 9e072a1.